variables.tf

```diff
@@ -10,6 +10,15 @@ variable "name" {
   default = "codebuild"
 }
 
+variable "cache" {
+  default = "true"
+  description = "If cache is true, create an S3 bucket for storing codebuild cache inside"
+}
+variable "cache_timeout" {
```

Comment:
Wonder if maybe `cache_expiration_days` would be better. See https://github.com/cloudposse/terraform-aws-s3-log-storage/blob/master/variables.tf#L89

Reply:
good idea.

I'll update the Readme as well.
Thanks for the review and feedback.